package.json

```diff
@@ -11,26 +11,22 @@
   ],
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build-storybook": "lerna run build-storybook --stream",
-    "develop": "lerna run develop --parallel --ignore @strapi/design-system-docs",
-    "build": "lerna run build --stream",
+    "develop": "lerna run develop --stream",
+    "build": "lerna run build",
```

Comment:
Does it make sense because this would always build everything (e.g. including the website, wouldn't it?)

Reply:
not the website at the moment because it's not part of the `monorepo` but it's good for SB :)

Reply:
Which is what we're already doing